introspection-engine/connectors/sql-introspection-connector/src/commenting_out_guardrails.rs

```diff
@@ -17,6 +18,20 @@ pub fn comment_out_unhandled_models(datamodel: &mut Datamodel) {
         }
     }
 
+    //fields with an empty name
+    for model in &mut datamodel.models {
+        for field in &mut model.fields {
+            if field.name == "".to_string() {
```

Comment:
alternatively `field.name.is_empty()` but it's good as it is :+1:

Reply:
I'll clean it up in a later PR.